saleor/graphql/channel/tests/mutations/test_channel_update.py

```diff
@@ -701,3 +714,39 @@ def test_channel_update_mutation_duplicated_warehouses(
     assert errors[0]["field"] == "warehouses"
     assert errors[0]["code"] == ChannelErrorCode.DUPLICATED_INPUT_ITEM.name
     assert errors[0]["warehouses"] == [add_warehouse]
+
+
+def test_channel_update_order_settings_manage_orders(
```

Comment:
Can you also add a test with manage_orders and input that contains more that single field to update? (PermissionDenied)

Reply:
Also, separate test for calling the mutation as an App